l1-contracts/contracts/dev-contracts/L1NullifierDev.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.24;
+
+import {L1Nullifier, IBridgehub} from "../bridge/L1Nullifier.sol";
+
+contract L1NullifierDev is L1Nullifier {
+    constructor(
+        IBridgehub _bridgehub,
+        uint256 _eraChainId,
+        address _eraDiamondProxy
+    ) L1Nullifier(_bridgehub, _eraChainId, _eraDiamondProxy) {}
+    function setL2LegacySharedBridge(uint256 _chainId, address _l2Bridge) external {
```

Comment:
please make a space between this function and constructor

Reply:
Why did the linter not complain?